### PR TITLE
Fixes #560

### LINKED ISSUE #560: Error code missing
The following error code: 
- `index_not_found` 

is missing in the `errors.yaml` file resulting in an absence of this error code in the documentation: https://docs.meilisearch.com.

It should be added with a clear and concise description.

Mentionned here: meilisearch/meilisearch-js#638

### PR DESCRIPTION
**Pull request by CreaTorAleXander**
Hey, here the updated error.yaml.

### CODE REVIEW

**Review by tpayet — APPROVED** [review]
LGTM :D thank you @CreaTorAleXander, I'll let @fharper or @react-learner merge this if they approve ofc

**Review by fharper — APPROVED** [review]
Awesome, thanks @CreaTorAleXander !

**Pull request merged**
